README.md

```diff
@@ -6,771 +6,51 @@ A collection of github repositories related to android modding.
 - [x] Breakup everything into more categories and adjust accordingly 
 - [x] Add descriptions for each project 
 - [ ] Remove unusable projects and duplicates from the list (*in progress*)
-- [ ] Make everything look nicer (*in progress*)
+- [X] Make everything look nicer
+- [ ] Sort stuff Metzger100 didn't know where to put
```

Comment:
Reword to something like "Sort miscellaneous projects"

Reply:
Done